changelog/23802.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+core/mounts: Reading an "auth" mount using "sys/internal/ui/mounts/" when filter paths are enforced returns 500 error code from the secondary
```

Comment:
Should this contain the "fix" text rather than what the original bug was?

Reply:
Good call :)